static/js/settings.js

```diff
@@ -53,6 +55,38 @@ const renderExtraIDs = () => {
   root.appendChild(extraIDsNode);
 };
 
+const addBlacklistedID = (blacklistedID) => {
```

Comment:
Reuse the code for handling the extraId, maybe with an extra param. So you can prevent code duplication.

Reply:
Done